_posts/en/newsletters/2020-08-12-newsletter.md

```diff
@@ -0,0 +1,215 @@
+
+- [LND 0.11.0-beta.rc2][lnd 0.11.0-beta] is a release candidate for the
+  next major version of this project.  It allows accepting [large
+  channels][topic large channels] (by default, this is off) and contains
+  numerous improvements to its backend features that may be of interest
+  to advanced users (see the [release notes][lnd 0.11.0-beta]).
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #18991][] changes the way that Bitcoin Core responds to
+  `getaddr` requests from its peers. Bitcoin nodes learn about potential new
+  peers to connect to in two different ways. Firstly, each node periodically
+  announces its own network address to its peers. Those peers re-announce the
+  address to some of their own peers, who re-announce to their peers, and so on,
+  propagating the address around the network. Secondly, a node can explicitly
+  request addresses from a peer using a `getaddr` message. To prevent a peer
+  from learning about all of the addresses in its address manager, Bitcoin Core
+  only provides a subset of those addresses when a peer sends it a `getaddr`
+  message, and only responds to a single `getaddr` message from each peer.
+  However, an adversary may still be able to learn all the addresses a peer knows
+  by making multiple connections to that peer and sending `getaddr` messages on
+  each connection. The adversary could potentially use that information to
+  fingerprint the peer or infer network topology. To prevent that, this PR
+  changes Bitcoin Core's behaviour to cache responses to `getaddr` requests and
+  provide the same response to all peers who request addresses in a 24-hour
+  rolling window.
+
+- [Bitcoin Core #19620][] prevents Bitcoin Core from re-downloading
+  unconfirmed segwit transactions that attempt to spend non-standard
+  UTXOs.  A non-standard UTXO is one that uses a currently discouraged
+  feature, such as using v1 segwit outputs before those have been
+  enabled on the network by a soft fork like [taproot][topic taproot].
+  This helps address a concern described in [Newsletter #108][news108
+  wtxid] about a potential taproot activation: nodes that don't upgrade
+  for the soft fork won't accept unconfirmed taproot transactions, so
+  they might end up downloading and rejecting the same unconfirmed
+  taproot transactions over and over again.  That won't happen to any
+  node running this patch, allowing backports of this patch to serve as
+  an alternative to [backporting the wtxid relay feature][Bitcoin Core
+  #19606], which would also prevent that wasted bandwidth.
+
+- [C-Lightning #3909][] the `listpays` RPC now returns a new
```

Comment:
This isn't a complete sentence. Consider changing to "C-Lightning 3909 changes the `listpays` RPC to return..."